### DISCUSSION TITLE
Don't resize window when dragged by the mouse.

### DISCUSSION THREAD

**Discussion by lexicalunit** [rxhanson/Rectangle#572] [Ideas]
This is driving me crazy. Sometimes I need to move a window a bit to peak behind it or so that I can see some content behind it at the same time and then BOOM my window suddenly distorts into some seemingly random width and height. Please no! I **never** want window movement to imply window resizing. If I wanted the window resized I'd do it manually via mouse or Rectangle commands.

**Comment by lexicalunit** [comment]
Wait, I think `defaults write com.knollsoft.Rectangle unsnapRestore -int 2` may have resolved this for me!

**Comment by lexicalunit** [comment]
Nope. Never mind. It's still doing it 😢

**Reply by rxhanson** [reply]
Did you restart the app after executing the terminal command?

**Reply by lexicalunit** [reply]
Yep I think that was it 👍🏻

**Comment by lexicalunit** [comment]
Waaaait. Maybe I just needed to restart Rectangle? I think it's fixed!
